### ISSUE TITLE
Remove 'blog' from the top navigation bar

### ISSUE THREAD

**Issue by MSchirp** [jetlej/HE-6.0-Bugs#95]
Since now almost all our content will be our own blog posts, I don't think it adds anything. Or am I missing something?

**Comment by jetlej** [comment]
Agreed.

**Comment by jetlej** [comment]
@MSchirp - We still need some nav item to allow people to see what shows under Blog. What do you suggest we replace it with?

**Comment by MSchirp** [comment]
Could you rephrase that question? not sure what you're asking

**Comment by jetlej** [comment]
@MSchirp - Look at the dropdown from the 'Blog' menu item on http://hiexistence.com. We still need a nav item for that content. So do you want to keep blog, or change it to something else?

**Comment by MSchirp** [comment]
What do you think of HEniverse?

**Comment by jetlej** [comment]
HE-verse? I don't think it's that clear.

**Comment by MSchirp** [comment]
No, I actually agree. @JonBrooksX @lostboyevsky Any suggestions guys?

**Comment by JonBrooksX** [comment]
I like your thinking with the outside-the-box-name, guys. But yeah, agreed that it would only be immediately clear for the true fans. Blog works for now. But here are other ideas: 

- Library
- Vault
- Crypt
- Archive 
- Inventory 
- Syllubus 
- Infotainment 
- Core

Any of those resonate with you?

**Comment by MSchirp** [comment]
I like Vault, Crypt and Archive :)

**Comment by jetlej** [comment]
How about 'Filter'? I also like Vault and Archive

**Comment by JonBrooksX** [comment]
I'm not too keen on filter. 

I vote for archive or "Best of" 

what does everyone else vote for? 

@MSchirp @jetlej @lostboyevsky

**Comment by MSchirp** [comment]
I love Archive

**Comment by jetlej** [comment]
Okay let's go with Archive

**Comment by jetlej** [comment]
as a reminder, this is something ya'll will be able to change yourself in wp-admin > Appearance > Menus
